CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Change Log
 
+## v0.30.0
+
+Released on 2021-10-01
+
+### Major Changes
+
+
+* new: add `--k8s-node` command-line options, which allows filtering by a node when requesting metadata of pods to the K8s API server [[#1671](https://github.com/falcosecurity/falco/pull/1671)] - [@leogr](https://github.com/leogr)
+* new(outputs): expose rule tags and event source in gRPC and json outputs [[#1714](https://github.com/falcosecurity/falco/pull/1714)] - [@jasondellaluce](https://github.com/jasondellaluce)
+* new(userspace/falco): add customizable metadata fetching params [[#1667](https://github.com/falcosecurity/falco/pull/1667)] - [@zuc](https://github.com/zuc)
+
+
+### Minor Changes
+
+* update: bump driver version to 3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4 [[#1744](https://github.com/falcosecurity/falco/pull/1744)] - [@zuc](https://github.com/zuc)
+* docs: clarify that previous Falco drivers will remain available at https://download.falco.org and no automated cleanup is run anymore [[#1738](https://github.com/falcosecurity/falco/pull/1738)] - [@leodido](https://github.com/leodido)
+* update(outputs): add configuration option for tags in json outputs [[#1733](https://github.com/falcosecurity/falco/pull/1733)] - [@jasondellaluce](https://github.com/jasondellaluce)
+* update: driver version is 5727c456ce22f3c1da8c3b1d7d6b6937a9b2126b now [[#1734](https://github.com/falcosecurity/falco/pull/1734)] - [@zuc](https://github.com/zuc)
```

Comment:
The double driver version update can be very confusing for users and brings no valuable info to them.

@leogr I suggest to go into this PR and insert NONE in the release-block portion of the body 


Then, this change log would need to be regenerated and (force) pushed on this branch

Reply:
@leodido Done! And thank you for having caught that. 

@araujof could we regenerate the changelong?

Reply:
@leogr will do! Thanks!